packages/elements-core/src/components/Docs/HttpService/HttpService.tsx

```diff
@@ -64,7 +63,7 @@ const HttpServiceComponent = React.memo<HttpServiceProps>(({ className, data, he
       ) : (
         <Box mb={10}>
           {description}
-          <PoweredByLink source={data.name ?? 'no-title'} pathname={pathname} packageType="elements" headless />
+          {pathname && <PoweredByLink source={data.name ?? 'no-title'} pathname={pathname} packageType="elements" headless />}
```

Comment:
Does this mean the powered by link is sometimes hidden?

Reply:
Yes, in `platform-internal` usages.

Reply:
...and when we forget to provide proper props, so I will actually add e2e test that it is actually visible in API.

I don't remember if it was supposed to be visible in StoplightProject? @philsturgeon

Reply:
Yeah we want powered by to show up on stoplightproject too.

To make it easier to read, there could be a hidePoweredBy variable or something like that, if its purely about platform internal. We might have an option to hide powered by if theyre on a certain plan too, so it would generally be a handy feature to have so long as its not a simple boolean in the control of the user.